src/ui/ViewModels/CourseRunningStatus.cs

```diff
@@ -6,6 +6,8 @@ public enum CourseRunningStatus
     {
         Running,
         NotRunning,
-        New
+        New,
+        NeedsAttention,
```

Comment:
What about calling the enum something that reflects the state in ucas rather than the effect of that? I'm not clear from this name what it means in terms of what's the state in the ucas system.

Reply:
This enum is a very over loaded usage, its is used for both denote a course as well as a site